patches/099-sqlite_api_3_26_0.patch

```diff
@@ -185,21 +1143,8 @@ index 3bc99ed2924b1756e7359e4be29f0702b1dac962..115895b8501992cc8ca4bd66281e0e5a
  
    // These all take a 0-based argument index.
    bool ColumnBool(int col) const;
-diff --git a/third_party/WebKit/Source/modules/webdatabase/DatabaseAuthorizer.cpp b/third_party/WebKit/Source/modules/webdatabase/DatabaseAuthorizer.cpp
-index f9e5d18cc9c0b59a8c1d8488f340a5de2a539569..f2f0b3e2cdc51f3f86d30ce9159acc89198e8881 100644
---- a/third_party/WebKit/Source/modules/webdatabase/DatabaseAuthorizer.cpp
-+++ b/third_party/WebKit/Source/modules/webdatabase/DatabaseAuthorizer.cpp
-@@ -72,7 +72,7 @@ const FunctionNameList& WhitelistedFunctions() {
-       ({
-           // SQLite functions used to help implement some operations
-           // ALTER TABLE helpers
--          "sqlite_rename_table", "sqlite_rename_trigger",
-+          "sqlite_rename_column", "sqlite_rename_table", "sqlite_rename_test",
```

Comment:
why was this patch removed ? Its a necessary change for 3.26.

Reply:
it was an error on my part. I've restored it in efa6c4c